espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/SetClipboard.kt

```diff
@@ -35,6 +34,13 @@ class SetClipboard : RequestHandler<SetClipboardParams, Void?> {
     @Throws(AppiumException::class)
     override fun handleInternal(params: SetClipboardParams): Void? {
         params.content ?: throw InvalidArgumentException("The 'content' argument is mandatory")
+
+        // FIXME: Can be null while contentType should be ClipboardDataType.PLAINTEXT by default
+        if (params.contentType == null
```

Comment:
so it cannot be null anymore, or it still can?

Reply:
It still can if a request had no 'plaintext' or 'PLAINTEXT' like 'image' in content Type attribute

Reply:
this is weird..., perhaps we just don't know how GSON works

Reply:
Yeah...
When I used Gson, non-null did not work. I remember such discussion..

Reply:
I found some articles about GSON such as:
https://proandroiddev.com/most-elegant-way-of-using-gson-kotlin-with-default-values-and-null-safety-b6216ac5328c

I also found a Japanese article decoding the generated code into Java.
According to them, we cannot handle properly for non-null (default) value in some case in Kotlin.
So, we should like https://github.com/appium/appium-espresso-driver/pull/478/files#diff-88e41c77db8f06054ff5887d7d6e3031R27

It worked fine.

error example:
```
Selenium::WebDriver::Error::InvalidArgumentError: io.appium.espressoserver.lib.handlers.exceptions.InvalidArgumentException: Only [plaintext] content types are supported. 'image' is given instead
        at io.appium.espressoserver.lib.model.ClipboardDataType$Companion.invalidClipboardDataType(ClipboardDataType.kt:14)
        at io.appium.espressoserver.lib.model.SetClipboardParams.getContentType(SetClipboardParams.kt:33)
        at io.appium.espressoserver.lib.handlers.SetClipboard.handleInternal(SetClipboard.kt:40)
        at io.appium.espressoserver.lib.handlers.SetClipboard.handleInternal(SetClipboard.kt:31)
        at io.appium.espressoserver.lib.handlers.RequestHandler$DefaultImpls.handle(RequestHandler.kt:28)
        at io.appium.espressoserver.lib.handlers.SetClipboard.handle(SetClipboard.kt:31)
        at io.appium.espressoserver.lib.handlers.SetClipboard.handle(SetClipboard.kt:31)
        at io.appium.espressoserver.lib.http.Router.route(Router.kt:216)
        at io.appium.espressoserver.lib.http.Server.serve(Server.kt:53)
        at fi.iki.elonen.NanoHTTPD$HTTPSession.execute(NanoHTTPD.java:945)
        at fi.iki.elonen.NanoHTTPD$ClientHandler.run(NanoHTTPD.java:192)
        at java.lang.Thread.run(Thread.java:764)
from InvalidArgumentError: io.appium.espressoserver.lib.han
```